### ISSUE TITLE
🐛 Bug: [installation.md] Clicking on "Run locally using Docker" is not navigating to the section 

### ISSUE THREAD

**Issue by AQIB-NAWAB** [json-schema-org/website#1217]
### Describe the bug

It's installation.md files in which when we click on **Run locally using Docker** but it is not navigating to the section as it should do 
[Screencast from 2024-12-27 16-13-23.webm](https://github.com/user-attachments/assets/5acb20fa-de2f-42e1-b939-526d6a997143)

one thing more i am getting error while setting the codebase using docker 
Here is the error 
![image](https://github.com/user-attachments/assets/81404042-b064-4d48-a9d1-6b9f265436dd)

### Steps To Reproduce

1. Go to the https://github.com/json-schema-org/website/blob/main/INSTALLATION.md#prerequisites
2. click on run it local using docker

### Expected Behavior

it should navigate to the docker section

### Screenshots

![image](https://github.com/user-attachments/assets/81404042-b064-4d48-a9d1-6b9f265436dd)

### Device Information [optional]

```markdown
- OS: Ubuntu
- Browser: Firefox
```

### Are you working on this issue?

Yes

### Do you think this work might require an [Architectural Decision Record (ADR)]? (significant or noteworthy)

No

**Comment by abhayymishraa** [comment]
@AQIB-NAWAB  This issue is already assigned.
see this #1196

**Comment by AQIB-NAWAB** [comment]
@abhayymishraa 
okk thanks for clarification but what about navigation issue 
Here is video 
[Screencast from 2024-12-27 16-13-23.webm](https://github.com/user-attachments/assets/5acb20fa-de2f-42e1-b939-526d6a997143)

**Comment by abhayymishraa** [comment]
yeah that was the good bug to take i think and also i'm not the  maintainer here

**Comment by abhayymishraa** [comment]
although you can close the issue!!!

**Comment by DhairyaMajmudar** [comment]
Thanks, @AQIB-NAWAB for the issue, the error screenshots attached are worked upon in this issue https://github.com/json-schema-org/website/issues/1196. 

Although the installation.md file does contain a broken link, which can be worked upon under this issue.

**Comment by shantanu2107** [comment]
Hi @DhairyaMajmudar, can you assign this issue to me, if it is not taken by anyone?

**Comment by AQIB-NAWAB** [comment]
@DhairyaMajmudar I will drop PR soon today.
